sql-composer/src/parser.rs

```diff
@@ -974,7 +1000,7 @@ mod tests {
     fn it_parses_quoted_bindvar() {
         let input = "':bind(varname)'blah blah blah";
 
-        let out = bindvar(Span::new(input.into())).expect("expected Ok from bindvar");
+        let out = bindvar_item(Span::new(input.into())).expect("expected Ok from bindvar");
```

Comment:
expect errors should refer to 'bindvar_item' rather than 'bindvar' in these tests.